naked-objects/dom/src/test/java/domainapp/dom/modules/simple/SimpleObjectTest.java

```diff
@@ -30,24 +30,18 @@
   public void setUp() throws Exception {
     simpleObject = new SimpleObject();
   }
-
-  /**
-   * Test for Names for SimpleObjects
-   */
-  public static class Name extends SimpleObjectTest {
-
-    @Test
-    public void happyCase() throws Exception {
-      // given
-      String name = "Foobar";
-      assertThat(simpleObject.getName()).isNull();
-
-      // when
-      simpleObject.setName(name);
-
-      // then
-      assertThat(simpleObject.getName()).isEqualTo(name);
-    }
+  
+  @Test
+  public void testName() throws Exception {
+    // given
+    String name = "Foobar";
+    assertThat(simpleObject.getName()).isNull();
```

Comment:
We should use `assertNull(simpleObject.getName())` here.

Reply:
What I feel is, it's good that AssertJ was used for assertions. It makes assertions readable, I was planning to either use Hamcrest or AssertJ in all modules. So I think we should keep assertion as is. And I will raise other issue to discuss assertion options.

Reply:
What I feel is, it's good that AssertJ was used for assertions. It makes assertions readable, I was planning to either use Hamcrest or AssertJ in all modules. So I think we should keep assertion as is. And I will raise other issue to discuss assertion options.